src/Exthost/Request.re

```diff
@@ -80,6 +80,37 @@ module ExtensionService = {
   };
 };
 
+module LanguageFeatures = {
+  let provideCompletionItems =
+      (
+        ~handle: int,
+        ~resource: Uri.t,
+        ~position: OneBasedPosition.t,
+        ~context: CompletionContext.t,
+        client,
+      ) => {
+    let parser = _ => {
```

Comment:
TODO: Implement the parser. The data model for completion items has actually changed significantly since the previous iteration - it uses a 'condensed' JSON model to reduce bytes-over-wire and to parse (which is nice for perf!)